tools/demikernel_ci.py

```diff
@@ -71,7 +72,7 @@ def run_pipeline(
     # STEP 4: Run integration tests.
     if test_integration:
         if status["checkout"] and status["compile"]:
-            if libos == "catnap" or libos == "catnapw" or libos == "catloop" or libos == "catnip" or libos == "catpowder":
+            if libos == "catnap" or libos == "catloop" or libos == "catnip" or libos == "catpowder":
```

Comment:
How about using the `libos in [...]` syntax here?

Reply:
Oh yeah!